src/test/kotlin/com/atlassian/performance/tools/jiraactions/api/scenario/BackupConfigurationIT.kt

```diff
@@ -0,0 +1,86 @@
+package com.atlassian.performance.tools.jiraactions.api.scenario
+
+import com.atlassian.performance.tools.dockerinfrastructure.api.browser.DockerisedChrome
+import com.atlassian.performance.tools.dockerinfrastructure.api.jira.JiraCoreFormula
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.measure.output.CollectionActionMetricOutput
+import com.atlassian.performance.tools.jiraactions.api.memories.User
+import com.atlassian.performance.tools.jiraactions.api.memories.UserMemory
+import com.atlassian.performance.tools.jiraactions.api.page.isElementPresent
+import com.atlassian.performance.tools.jiraactions.api.w3c.DisabledW3cPerformanceTimeline
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import org.openqa.selenium.By
+import java.time.Clock
+import java.util.*
+
+class BackupConfigurationIT {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    /**
+     * During the test, you can connect to WebDriver by the VNC viewer.
+     * DockerisedChrome opens the port 5900 so executing command
+     * docker ps
+     * will show you which port on your host is open and mapped to 5900 on the container.
+     */
+    @Test
```

Comment:
Starting another convo for the test duration.
Here's my local report: 
![image](https://user-images.githubusercontent.com/1194704/62789850-1af5fe80-baca-11e9-90e5-074f76472996.png)

We should either start merging the integration tests or bump the `travis_wait`

Reply:
I will bump the `travis_wait` maybe we should bump to 7 minutes to avoid flakiness?
or there is no such danger?

Reply:
7 minutes sound good